lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -227,7 +238,16 @@ import Control.Arrow
 import Control.Monad
     ( forM, forM_, replicateM )
 import Control.Monad.Random
-    ( MonadRandom (..), Random (randomR, randomRs), random, randoms )
+    ( MonadRandom (..)
+    , Rand
+    , Random (randomR, randomRs)
```

Comment:
👍🏻 I also like this style of import (where only the functions needed are imported explicitly).